caps/static/caps/scss/backgrounds.scss

```diff
@@ -5,3 +5,21 @@
 .bg-orange-gradient {
     background-image: linear-gradient(45deg, $color-ceuk-red, $color-ceuk-orange) !important;
 }
+
+$theme-colours: (
```

Comment:
Why define our own `$theme-colours` in this file, when there’s already a similarly named and almost identical `$theme-colors` in `caps/variables.scss`?

I’d just drop this `$theme-colours` declaration, and change the `@each` to loop over `$theme-colors` instead.

Reply:
Good one, I just added a fixup that fixes the issue.